hicommonwealth/commonwealth
# 6684 stake admin enablement

## Link to Issue
Closes: #6684 (this depends on @mzparacha 's #7394 )

## Description of Changes
- Adds staking community steps into enablement flow

## Test Plan
- Enable `FLAG_EXISTING_COMMUNITY_STAKE_INTEGRATION_ENABLED` in `.env`
- Visit `/:scope/manage/integrations` page of any community without stake enabled you are an admin of, verify you see the stake section with relevant UI, verify that clicking the button redirects you to `/:scope/manage/integrations/stake`
- Ensure you can go through the entire staking process via the admin integration selection
- Visit `/:scope/manage/integrations` page of any community that cannot have stake enabled yet (any chain with ethChainId not included in [these]
